### ISSUE TITLE
Get Alexa working on Rpi without further use of VNC

### ISSUE THREAD

**Issue by Eddd97** [alexa/alexa-avs-sample-app#444]
Hi! Is there any method I can make Alexa work without using VNC often. As far as I know the Bearer Token expires after some time and it needs to be renewed autehnticating again in browser and also Sensory will return "iteration limit exceeded" after a day or two.  Is there any workaround to start alexa at boot time or at least after the first browser authorization to get rid of need to use VNC?

**Comment by elstryan** [comment]
Hi Eddd97, 
For this there are a couple of options: 
1) You can use an android or iOS mobile companion app for authorization. The steps are outlined [here for this project](https://github.com/alexa/alexa-avs-sample-app/wiki/Mac#5b---build-and-run-an-android-companion-app-authentication) and the process is described in our [public documentation](https://developer.amazon.com/public/solutions/alexa/alexa-voice-service/docs/authorizing-your-alexa-enabled-product-from-an-android-or-ios-mobile-app). If you use a mobile application for authentication then your application will be loaded with a refresh token which can be used indefinitely without having the user re authenticate. 
2) Right now the companion service is setup to use an implicit grant. You can modify how the sample app and the companion service interact to use authorization code grant instead. The process is documented here: https://developer.amazon.com/public/solutions/alexa/alexa-voice-service/docs/authorizing-your-alexa-enabled-product-from-a-website

**Comment by Eddd97** [comment]
Hello again! I tried using the first option by building the companion app on Android for Authorization, but I  am facing a weird issue: I get "java.security.cert.CertPathValidatorException: Trust anchor for certification path not found" on Android app. I tried creating new certs, adding my phone's ip to ssl.cnf (I installed the app on my phone), change the provisioningMethod to companionApp, but I keep getting this error. What am I missing?
